DOCKER.md

```diff
@@ -35,12 +35,13 @@ docker run avenga/couper run -watch -p 8081
 | COUPER_FILE                          | `couper.hcl` | Path to the configuration file. |
 | COUPER_WATCH                         | `false` | Set to `true` to watch for configuration file changes. |
 | COUPER_WATCH_RETRY_DELAY             | `500ms` | Delay duration before next attempt if an error occurs. |
-| COUPER_WATCH_RETRIES                 | `5` | Maximal retry count for configuration reloads which could not bind the configured port. |
-| COUPER_DEFAULT_PORT                  | `8080` | Sets the default port to the given value and does not override explicit `[host:port]` configurations from file. |
+| COUPER_WATCH_RETRIES                 | `5`     | Maximal retry count for configuration reloads which could not bind the configured port. |
+| COUPER_DEFAULT_PORT                  | `8080`  | Sets the default port to the given value and does not override explicit `[host:port]` configurations from file. |
 | COUPER_XFH                           | `false` | Global configurations which uses the `Forwarded-Host` header instead of the request host. |
 | COUPER_HEALTH_PATH                   | `/healthz` | Path for health-check requests for all servers and ports. |
+| COUPER_HTTPS_DEV_PROXY               | `""`    | List of tls port mappings to define the tls listen port and the target one. A self-signed certificate will be generated on the fly based on given hostname. |
```

Comment:
OVERALL:

...generated on the fly based on given hostname and is valid for 24 hours.

?

Reply:
I have added an expired check instead of documenting the certificate duration.